tests/test_generated.py

```diff
@@ -10,15 +10,20 @@
 import json
 import pytest
 
+generated_tests = {
+    test.path: test
+    for test in generate_tests()
+}
+
 
 @pytest.fixture(scope="session")
 def bq():
     return bigquery.Client()
 
 
-@pytest.fixture(params=list(generate_tests()))
+@pytest.fixture(params=list(generated_tests.keys()))
 def generated_test(request):
-    return request.param
+    return generated_tests[request.param]
```

Comment:
this makes `pytest` report the generated test path in test names for easier debugging, which I needed for verifying that all the tests were still being run